packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -42,25 +60,32 @@ export const PanelChrome: React.FC<PanelChromeProps> = ({
   const headerStyles: CSSProperties = {
     height: headerHeight,
   };
-
+  const itemStyles: CSSProperties = {
```

Comment:
Yes, it will be used for each icon next to the title, but I removed them all temporarily. I removed most of my first changes and am now incrementally adding what we agreed on here in this PR, so that it will be the only functionality present at the moment.